sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/v2ResolutionPlans.scala

```diff
@@ -54,7 +54,7 @@ case class UnresolvedView(
     multipartIdentifier: Seq[String],
     commandName: String,
     allowTemp: Boolean,
-    hint: Boolean) extends UnresolvedLeafNode
+    suggestAlternative: Boolean) extends UnresolvedLeafNode
```

Comment:
to be consistent with `UnresolvedTable`, shall we give it a default value (false) as well?

Reply:
The above modifications are submitted as following PR, and I will do it today.

Reply:
A new follow up pr: https://github.com/apache/spark/pull/42917.